smbios.adoc

```diff
@@ -16,24 +16,23 @@ NOTE: The structures and fields in this section are defined in a manner consiste
 [%header, cols="5,25"]
 |===
 | ID#     ^| Requirement
-| `SMBIOS_010` | A Baseboard/Module Information (Type 02) structure SHOULD be implemented.
-| `SMBIOS_020` | A System Enclosure/Chassis (Type 03) structure SHOULD be implemented.
+| `SMBIOS_010` | The essential structures and data from Table 4, Section 6.2, along with the conformance guidelines in Annex A of the cite:[SMBIOS] specification, MUST be implemented if hardware support is present.
```

Comment:
Annex A is informative, i.e. non-normative, so let's instead refer to Annex A inside our own smbios guidelines and drop it from here

Reply:
noted